client/src/components/technologies/technology-picker/TechnologyPicker.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="technology-picker">
+    <div class="technology-picker-search input">
+      <img
+        class="input-image"
+        src="../assets/search.svg">
+      <input
+        class="input-box with-image"
+        type="text"
+        placeholder="Search technology"
+        v-model = "search">
+    </div>
+    <chip
+      v-for="technology in technologies"
+      :key="technology.id"
+      :name="technology.name"></chip>
+  </div>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+  import Chip from '../../common/Chip/Chip.vue';
+  import * as types from '../../../store/modules/technologies/technology-types';
+  import {ITechnology} from "../../../shared/interfaces/ITechnology";
+
+  export default Vue.extend({
+    data() {
+      return {
+        search: ''
+      }
+    },
+    components: {
+      Chip
+    },
+    created() {
+      this.$store.dispatch(types.FETCH_TECHNOLOGIES);
+    },
+    computed: {
+      technologies(): ITechnology[] {
+        return this.$store.state.technologies.technologies;
+      }
+    },
+    watch: {
+      search: function (val) {
+        // TODO implement filtering through technologies
+        // best practice? 
+        console.log(val);
```

Comment:
Just use @change on the input instead

Reply:
we have to filter computed technologies that come from store, should i create local data property filtered projects and filter it, otherwise have to filter computed